rosa_sts_managed_oidc_no_modules/variables.tf

```diff
@@ -0,0 +1,191 @@
+variable "rosa_openshift_version" {
+  type        = string
+  default     = "4.13"
+  description = "Desired version of OpenShift for the cluster, for example '4.1.0'. If version is greater than the currently running version, an upgrade will be scheduled."
+}
+
+# Account Roles
+variable "account_role_prefix" {
+  type    = string
+  default = "mobb"
+}
+
+variable "account_role_policies" {
+  description = "account role policies details for account roles creation"
+  type = object({
+    sts_installer_permission_policy             = string
+    sts_support_permission_policy               = string
+    sts_instance_worker_permission_policy       = string
+    sts_instance_controlplane_permission_policy = string
+  })
+  default = null
+}
+
+# Operator Roles
+variable "operator_role_prefix" {
+  type    = string
+  default = "mobbtf"
+}
+
+variable "operator_role_policies" {
+  description = "operator role policies details for operator roles creation"
+  type = object({
+    openshift_cloud_credential_operator_cloud_credential_operator_iam_ro_creds_policy = string
+    openshift_cloud_network_config_controller_cloud_credentials_policy                = string
+    openshift_cluster_csi_drivers_ebs_cloud_credentials_policy                        = string
+    openshift_image_registry_installer_cloud_credentials_policy                       = string
+    openshift_ingress_operator_cloud_credentials_policy                               = string
+    openshift_machine_api_aws_cloud_credentials_policy                                = string
+  })
+  default = null
+}
+
+variable "create_vpc" {
+  type        = bool
+  description = "Create custom VPC for ROSA cluster"
+}
+
+variable "managed_oidc" {
+  type        = bool
+  description = "Red Hat managed or unmanaged (Customer hosted) OIDC Configuration"
+  default     = true
+}
+
+# ROSA Cluster info
+variable "cluster_name" {
+  type        = string
+  description = "The name of the ROSA cluster to create"
+  default     = "poc-andyr"
```

Comment:
Same question here, do we want to specify non-user specific vars?

Reply:
No, I'll make it a prompt